niviz_rater/validation.py

```diff
@@ -0,0 +1,49 @@
+import yaml
+import json
+import os
+import yamale
+from yamale.validators import DefaultValidators, Validator
+from bids.config import get_option
+
+SCHEMAFILE = os.path.join(os.path.dirname(__file__), 'data/schema.yaml')
+
+
+def load_json(file):
+    with open(file, 'r') as f:
+        result = json.load(f)
+    return result
+
+
+def get_entities():
+    configfiles = [
+        load_json(file)['entities']
+        for file in get_option('config_paths').values()
+    ]
+    enumstrs = list()
```

Comment:
```suggestion
    enumstrs = []
```

square brackets are more of a standard convention :)

Reply:
Hard to shake the R in me hahaha